packages/js/components/src/experimental-select-control/menu.tsx

```diff
@@ -22,9 +29,25 @@ export const Menu = ( {
 	isOpen,
 	className,
 }: MenuProps ) => {
+	const [ boundingRect, setBoundingRect ] = useState< DOMRect >();
+	const selectControlMenuRef = useRef< HTMLDivElement >( null );
+
+	useEffect( () => {
+		if (
+			selectControlMenuRef.current &&
+			selectControlMenuRef.current?.parentElement
+		) {
+			setBoundingRect(
+				selectControlMenuRef.current?.parentElement.getBoundingClientRect()
```

Comment:
you don't need the `?` here either.

Reply:
Done as part of https://github.com/woocommerce/woocommerce/pull/34967/commits/8e895d4139b8369fd8c90943818bb20c7d10cf06